fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/api/JournalEntriesApiResourceSwagger.java

```diff
@@ -62,4 +64,77 @@ private PostJournalEntriesTransactionIdResponse() {
         @Schema(description = "1")
         public Long officeId;
     }
+
+    @Schema(description = "GetJournalEntriesTransactionIdResponse")
+    public static final class GetJournalEntriesTransactionIdResponse {
+
+        private GetJournalEntriesTransactionIdResponse() {
+
+        }
+
+        static final class EnumOptionType {
+
+            private EnumOptionType() {}
+
+            @Schema(example = "2")
+            public Long id;
+            @Schema(example = "accountType.asset")
+            public String code;
+            @Schema(example = "ASSET")
+            public String value;
+        }
+
+        static final class TransactionDetails {
+
+            private TransactionDetails() {}
+
+            @Schema(example = "2")
+            public Long transactionId;
+            public EnumOptionType transactionType;
+        }
+
+        static final class JournalEntryTransactionPageItems {
+
+            private JournalEntryTransactionPageItems() {}
+
+            @Schema(example = "1")
```

Comment:
Please add all the relevant fields from the JournalEntryData

Reply:
Done